kubectl/README.md

```diff
@@ -0,0 +1,31 @@
+# Tool builder: `gcr.io/cloud-builders/kubectl`
+
+This Container Builder build step runs
+[`kubectl`](https://kubernetes.io/docs/user-guide/kubectl-overview/).
+
+To use this builder, your [builder service account]
+(https://cloud.google.com/container-builder/docs/how-to/service-account-permissions)
+will need IAM permissions sufficient for the operations you want to perform. For
+typical read-only usage, the "Container Engine Viewer" role is sufficient. To
+deploy container images on a GKE cluster, the "Container Engine Developer" role
+is sufficient. Check the [GKE IAM page]
+(https://cloud.google.com/container-engine/docs/iam-integration) for details. 
+
+For most use, kubectl will need to be configured to point to a specific GKE
+cluster. You can configure the cluster by setting environment variables.
+
+    CLOUDSDK_COMPUTE_ZONE=<your cluster's zone>
```

Comment:
No, this are configurations for `gcloud container clusters get-credentials`, which does take them both as args. `kubectl` itself has no concept of GKE.